### DISCUSSION TITLE
How to upload larger files?

### DISCUSSION THREAD

**Discussion by CKylinMC** [nuxt/nuxt#28366] [Questions] [answered]
Hi! I'm learning to use nuxt to build a small app for my work. In my case, I have a file about 50mb, and I want to upload to the server built with nuxt. The problem is I always fails with :
```json
{
    "ErrorCode": "EntityTooLarge",
    "ErrorMessage": "payload size exceeds maximum allowed size (33554432 bytes)"
}
```
I have searched for it and tried to use formidable but not work, the event seems being handled even before my event handler.

My question is how to extend this limit?
Thanks!

**Comment by nowo** [comment, marked as answer]
I just wrote a nuxt plugin library, [https://github.com/nowo/nuxt-file-save](https://github.com/nowo/nuxt-file-save), you can try it out or look at the code in it, hope this helps!
I haven't tested it in a production environment yet, in a local development environment, uploading a 2GB file, it worked!
😆

**Reply by CKylinMC** [reply]
Thanks! I'll try!
